packages/core/build.mts

```diff
@@ -0,0 +1,21 @@
+import { build, defaultBuildOptions } from '@redwoodjs/framework-tools'
+
+// Build 'dist/bins'
+await build({
+  buildOptions: {
+    ...defaultBuildOptions,
+    entryPoints: ['src/bins/*'],
+    outdir: 'dist/bins',
+  },
+  metafileName: 'meta.bins.json',
+})
+
+// Build 'config'
+await build({
```

Comment:
Did we not build `config` before? Why do we need to now?

Reply:
Please see the other comment